doc/chapters/api/index.rst

```diff
@@ -24,88 +13,276 @@ Methods
 
 Ping
 ----
-Ping()
+Controls availability of com.pelagicore.SoftwareContainerAgent interface
+
+:Parameters:
+        - **None**
+
+:Return Value:
+        - **None** 
+
+:Example Usage:
+
+::
+
+        dbus-send --system --print-reply \
+        --dest=com.pelagicore.SoftwareContainerAgent \
+        /com/pelagicore/SoftwareContainerAgent \
+        com.pelagicore.SoftwareContainerAgent.Ping
```

Comment:
Maybe not, I thought that since the D-Bus parts was one on each line one might want to say what is ObjectPath, what is Interface etc. But perhaps one can expect the user to understand D-Bus since they are reading about the D-Bus API...